pkg/controller/migmigration/migrate.go

```diff
@@ -93,42 +80,52 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (int, er
 		return 0, err
 	}
 
-	migration.Status.SetCondition(migapi.Condition{
-		Type:     Running,
-		Status:   True,
-		Reason:   task.Phase.Name,
-		Category: Advisory,
-		Message:  RunningMessage,
-	})
-
 	// Result
-	migration.Status.Phase = task.Phase.Name
-	if task.Phase.Equals(WaitOnResticRestart) {
-		return 10, nil
-	}
-	if task.Phase.Final() {
+	migration.Status.Phase = task.Phase
+
+	// Succeeded
+	if task.Phase == Completed {
```

Comment:
similar concerns around merging phase refactoring

Reply:
The _Phase_ is still used by the controller internally and does not contradict the decision to represent the Status externally using conditions.